src/components/ActivityCard.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import styled from 'styled-components'
+import moment from 'moment'
+import 'moment/locale/de'
+moment.locale('de')
+
+const activityTypeIcon = {
+  Ride: '/assets/bike.svg',
+  Workout: '/assets/dumbbell.svg',
+  WeightTraining: '/assets/dumbbell.svg',
+  Run: '/assets/run.svg'
+}
+
+const StyledCard = styled.article`
+  background: var(--grey);
+  border-radius: 10px;
+  padding: 10px 15px;
+  margin-bottom: 20px;
```

Comment:
you can think about sorting your CSS in **alphabetical** order

Reply:
Will be done in coming sprint